draft-ietf-cdni-uri-signing.xml

```diff
@@ -631,76 +631,76 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
 
         <t>The method described in this section allows CDNs to use URI Signing
-        for HTTP Adaptive Streaming content (or other chunked content) without
+        for segmented content without
         having to include the Signed URIs in the manifest files themselves.</t>
       </section>
 
-      <section anchor="uri_signing_mechanism" title="Signed Token Chain mechanism">
-        <t>In order to allow for effective access control of HAS content, the
+      <section anchor="uri_signing_mechanism" title="Signed Token Renewal mechanism">
+        <t>In order to allow for effective access control of segmented content, the
         URI signing scheme defined in this section is based on a mechanism
-        through which subsequent HAS chunk requests can be chained together.
+        through which subsequent segment requests can be linked together.
         As part of the JWT validation procedure, the CDN can generate a new
         signed JWT that the UA can use to do a subsequent request. More specifically,
-        whenever a UA successfully retrieves an HAS chunk, it receives, in the
+        whenever a UA successfully retrieves a segment, it receives, in the
         HTTP 2xx Successful message, a signed JWT that it can use whenever it
-        requests the next chunk. As long as each signed JWT in such a Signed Token Chain
-        is correctly validated before a new one is generated, the chain is not
-        broken and the User Agent can successfully retrieve additional chunks.
-        Given the fact that with HAS protocols, it is usually not possible to
-        determine a priori which chunk will be requested next (i.e., to allow for
-        seeking within the content and for switching to a different quality level),
-        the Signed Token Chain uses the URI Pattern Container and/or the
+        requests the next segment. As long as each successive signed JWT
+        is correctly validated before a new one is generated, the model is not
+        broken and the User Agent can successfully retrieve additional segments.
+        Given the fact that with segmented protocols, it is usually not possible to
+        determine a priori which segment will be requested next (i.e., to allow for
+        seeking within the content and for switching to a different representation),
+        the Signed Token Renewal uses the URI Pattern Container and/or the
         URI Regular Expression Container scoping mechanisms in the URI Container
-        (sub) claim to allow the Signed Token Chain to be valid for more than one URL.</t>
+        (sub) claim to allow a signed JWT to be valid for more than one URL.</t>
 
-        <t>In order for this chaining of signed JWTs to work, it is necessary for
+        <t>In order for this renewal of signed JWTs to work, it is necessary for
         a UA to extract the signed JWT from the HTTP 2xx Successful message of an
```

Comment:
Does it have to be extracted from a 2xx message? It occurs to me that some CDNs might deliver these renewals during 304s or 302s or similar. Technically, a CDN could even renew a token on a 404, I think. This might could be more general.